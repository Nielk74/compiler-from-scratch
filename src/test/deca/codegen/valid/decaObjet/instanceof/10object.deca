// Description:
//    instanceof d'une classe avec Object
//
// Resultats:
//    a is instance of Object
//
// Historique:
//    cree le 21/06/2022

class A {}

{
    A a = new A();

    if (a instanceof Object) {
        println("a is instance of Object");
    } else {
        println("a is not instance of Object");
    }
}