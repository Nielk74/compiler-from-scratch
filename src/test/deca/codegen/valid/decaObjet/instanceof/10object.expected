a is instance of Object
